src/main/java/dinkplugin/notifiers/DeathNotifier.java

```diff
@@ -33,57 +52,94 @@ public void onActorDeath(ActorDeath actor) {
         if (isEnabled() && plugin.getClient().getLocalPlayer() == actor.getActor()) {
             this.handleNotify();
         }
+        lastTarget = new WeakReference<>(null);
+    }
+
+    public void onInteraction(InteractingChanged event) {
```

Comment:
if we want to be sure these interactions are attacks, I can switch this to `MenuOptionClicked` - but there's no enum for attack, so we'd have to match the option text, "Attack" (which I'm afraid some clients may rename)

Reply:
Doesn't osrs have servers with different languages where the text is translated? I don't think this approach will work well

Reply:
Joined a german world & not seeing the string translated - but, in principle, clients could have a plugin that renames it (e.g., for translation or just for fun) - so I agree this approach is more finnicky